var/spack/repos/builtin/packages/git/package.py

```diff
@@ -68,7 +75,7 @@ class Git(Package):
     depends_on("zlib")
 
     def install(self, spec, prefix):
-        env['LDFLAGS'] = "-L%s" % spec['gettext'].prefix.lib + " -lintl"
```

Comment:
do you mind explain why you change things like that? Especially when such changes are not related to the PR title.